### PR TITLE
Updating media extension docs for v2.0 release

### PR DESCRIPTION
**Pull request by addb**
- Added the description for new API (Player State Tracking)
- Added Constants for Player State Tracking
- Added synchronous API for createTracker and deprecated the createTracker with callback API.
- Removed configuration keys which are no longer being used by the SDK

### CODE REVIEW

**Comment by cacheung** [comment]
Do we need to mention that we are converting QOE from Double to Long?

**Comment by cacheung** [comment]
Can you also update the Media API  reference page.
Androd section of  "Tracking bitrate changes", there is a typo there.
It is using _heartbeat.trackEvent...
It supposed to be tracker.trackEvent...

File: using-mobile-extensions/adobe-media-analytics/adobe-media-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Release Notes
```
[the remark this answers is not in the document]
Comment (addb) [reply using-mobile-extensions/adobe-media-analytics/adobe-media-release-notes.md:1]: Will update it on the day we release.

**Comment by cacheung** [comment]
We don't have PREROLL_TRACKING_WAITING_TIME info in the ref doc.  We have that in MediaSDK.

File: using-mobile-extensions/adobe-media-analytics/media-api-reference.md [hunk 2]
```diff
@@ -495,6 +610,84 @@ NSDictionary *qoeObject = [ACPMedia createQoEObjectWithBitrate: 10000000
...
+{% endtab %}
+
+{% tab title="React Native" %}
+**JavaScript**
+
+#### createStateObject
+
+
+```jsx
+let qoeObject = ACPMedia.createStateObject("fullscreen");
```
Comment (praveek) [inline using-mobile-extensions/adobe-media-analytics/media-api-reference.md:688]: Rename qoeObject to playerStateObject

File: using-mobile-extensions/adobe-media-analytics/media-api-reference.md [hunk 3]
```diff
@@ -1470,6 +1957,66 @@ FOUNDATION_EXPORT NSString* _Nonnull const ACPAdMetadataKeySiteId;
...
+Defines some common Player State constants.
+
+{% tabs %}
+{% tab title="Android" %}
+```java
+public class MediaConstants {
+
+  public static final class PlayerState {
+    public static final String FULLSCREEN = "fullscreen";
+		public static final String PICTURE_IN_PICTURE = "pictureInPicture";
```
Comment (praveek) [inline using-mobile-extensions/adobe-media-analytics/media-api-reference.md:1987]: The indentation is not right here

File: using-mobile-extensions/adobe-media-analytics/media-api-reference.md [hunk 4]
```diff
@@ -1613,9 +2170,41 @@ typedef NS_ENUM(NSInteger, ACPMediaEvent) {
      * Constant defining event type for change in Bitrate
      */
     ACPMediaEventBitrateChange,
+    /**
+     * Constant defining event type for player State start
```
Comment (praveek) [inline using-mobile-extensions/adobe-media-analytics/media-api-reference.md:2174]: Constant defining event type for State start

File: using-mobile-extensions/adobe-media-analytics/media-api-reference.md [hunk 5]
```diff
@@ -1613,9 +2170,41 @@ typedef NS_ENUM(NSInteger, ACPMediaEvent) {
...
+     */
+    ACPMediaEventStateStart
+    /**
+     * Constant defining event type for  player State end
```
Comment (praveek) [inline using-mobile-extensions/adobe-media-analytics/media-api-reference.md:2178]: Constant defining event type for State end

**Comment by cacheung** [comment]
Good improvement for the documentation!

**Pull request merged**